cita-executor/core/src/libexecutor/block.rs

```diff
@@ -137,9 +138,12 @@ impl ExecutedBlock {
         }
     }
 
-    #[allow(unknown_lints, clippy::too_many_arguments)] // TODO clippy
-    pub fn apply_transaction(&mut self, t: &SignedTransaction, conf: &BlockSysConfig) {
+    pub fn apply_transaction(&mut self, t: &SignedTransaction, sys_config: &GlobalSysConfig) {
```

Comment:
Why change the `BlockSysConfig` to `GlobalSysConfig`

Reply:
I need to update `block_quota_limit` next.